python/oneflow/cuda/__init__.py

```diff
@@ -37,7 +37,7 @@ def current_device() -> int:
 
 def manual_seed_all(seed) -> None:
     r"""The documentation is referenced from:
-    https://pytorch.org/docs/stable/generated/torch.cuda.manual_seed_all.html?highlight=manual_seed_all
+    https://pytorch.org/docs/1.10/generated/torch.cuda.manual_seed_all.html?highlight=manual_seed_all
```

Comment:
```suggestion
    https://pytorch.org/docs/1.10/generated/torch.cuda.manual_seed_all.html
```

这类 hightlight 的感觉是不是可以顺便去掉，因为其实 torch 页面没这个功能，如果有的话其实也不好看。

应该是之前的作者，搜索地址，然后直接复制粘贴的搜索结果，所以引入了 highlight。其它hightlight也去掉吧。

Reply:
收到